src/script.js

```diff
@@ -206,6 +206,20 @@ function isScriptHashOutput (script) {
     buffer[22] === OPS.OP_EQUAL
 }
 
+function isWitnessPubKeyHashOutput (script) {
+  var buffer = compile(script)
+
+  return buffer.length === 22 &&
+    buffer[0] === OPS.OP_0
```

Comment:
```
+    buffer[1] === 0x14 
```
is worth checking in addition to checking the buffer size, so you know the data push is actually 20 bytes